pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -369,3 +378,19 @@ func (r *InferenceServiceReconciler) deleteExternalResources(isvc *v1beta1api.In
 	}
 	return nil
 }
+
+func (r *InferenceServiceReconciler) GetFailConditions(isvc *v1beta1api.InferenceService) string {
+	/*
+		logf.SetLogger(zap.New())
+		log := logf.Log.WithName("my_controller")
+		log.Info("GetFailConditions")
+		fmt.Println("my_GetFailConditions")
+	*/
+	msg := ""
+	for _, cond := range isvc.Status.Conditions {
+		if string(cond.Status) == "False" {
+			msg = fmt.Sprintf("%s, %s", msg, string(cond.Type))
```

Comment:
How will this be shown in the events if there is more than 1 condition?

Reply:
For example: " InferenceService [test-err-msg] is not Ready because of: IngressReady, PredictorReady "